plugins/available/jenv.plugin.bash

```diff
@@ -0,0 +1,8 @@
+cite about-plugin
+about-plugin 'load jenv, if you are using it'
+
+export JENV_ROOT="$HOME/.jenv"
+export PATH="$JENV_ROOT/bin:$PATH"
```

Comment:
Please use the `pathmunge` function here. Example usage: https://github.com/Bash-it/bash-it/blob/6ec761a994ad1a0c44cb21511acfbb458ed5469f/plugins/available/node.plugin.bash#L4